helm/prometheus-rules/templates/alerting-rules/up.all.rules.yml

```diff
@@ -33,7 +33,7 @@ spec:
       annotations:
         description: '{{`Cadvisor ({{ $labels.instance }}) is down.`}}'
         opsrecipe: kubelet-is-down/
-      expr: label_replace(up{app="cadvisor"}, "ip", "$1", "instance", "(.+):\\d+") == 0
```

Comment:
Was the `ip` label a problem for Mimir? Did you check no inhibition relies on it?

Reply:
I reverted this